README.md

```diff
@@ -0,0 +1,112 @@
+stm32flash
```

Comment:
We can need a README file, but pasting in the whole wiki page is maybe not right.